src/engine/renderer/glsl_source/computeLight_fp.glsl

```diff
@@ -101,6 +107,14 @@ void computeLight( vec3 lightDir, vec3 normal, vec3 viewDir, vec3 lightColor,
   NdotL = clamp( NdotL, 0.0, 1.0 );
 #endif
 
+#if defined(USE_REFLECTIVE_SPECULAR)
```

Comment:
See `else` on line 98, this part of the code never run with PBR.

Reply:
Maybe we should start indenting things in #ifdef, since it's too hard to see this stuff.

Reply:
I fully agree with the idea of indenting `#ifdef`, especially since all those `#ifdef` in glsl are workaround for the inability of GPUs to properly jump over useless instructions. If this was CPU code, that would be standard `if`.

That said, I have too many work-in-progress patches on those files in other branches to geopardize the various rebase that comes. So, I'm not going to indent `#ifdef` for the moment, probably not before the merge of #301.